02/app.js

```diff
@@ -17,13 +17,28 @@ const a = prompt("Podaj liczbę podstawy!");
 const n = prompt("Podaj liczbę wykładnika!");
 let number = 1;
 
+//while (number <= n) {
+//number++;
+
+//if (a > 1 && a < 9 && n > 1 && n < 9) {
+//let result = Math.pow(a, n);
+//console.log(result);
+//} else {
+//console.log("wprowadziłeś złe dane wybierz liczby z przedziału 1-9");
+//}
+//}
+
 while (number <= n) {
   number++;
+  let result = 0;
+ 
 
   if (a > 1 && a < 9 && n > 1 && n < 9) {
-    let result = Math.pow(a, n);
-    console.log(result);
-  } else {
-    console.log("wprowadziłeś złe dane wybierz liczby z przedziału 1-9");
+
+    
+      result = a ** n;
+     
+      console.log( result);
```

Comment:
Nadal to nie jest to co mamy zobaczyć w konsoli ;) zamiast samego `8` (przy podaniu 2 i 3) to powinno być `2 * 2 * 2 = 8`